robot_smach_states/examples/manipulation/grasp_benchmark/grasping_height_test.py

```diff
@@ -0,0 +1,77 @@
+#! /usr/bin/env python
+
+# System
+from collections import namedtuple
+
+# ROS
+import PyKDL as kdl
+import rospy
+
+# TU/e Robotics
+from robot_smach_states.manipulation import Grab
+from robot_smach_states.navigation import NavigateToSymbolic
+from robot_smach_states.util.designators import EntityByIdDesignator, UnoccupiedArmDesignator
+
+# Robot Skills
+from robot_skills import arms
+from robot_skills.hero_parts.hero_arm import HeroArm
+from robot_skills.get_robot import get_robot_from_argv
+from robot_skills.util.kdl_conversions import FrameStamped
+
+DZ = 0.1
+Z_MAX = 1.2
+
+X_ENTITY = 3.2
+Y_ENTITY = 1.55
+
+Result = namedtuple("Result", ["height", "result", "duration"])
+
+if __name__ == "__main__":
+
+    rospy.init_node("test_grasping")
+
+    robot = get_robot_from_argv(1)
+
+    # Navigate to the couch table
+    nav_designator = EntityByIdDesignator(robot=robot, id="couch_table")
```

Comment:
Why navigate to the couch table? If we are grasping a virtual entity anyway why don't we just place that entity in front of the current position of the robot?

Reply:
Good question. The supporting entity (in this case, the couch table) affects the way the robot can position itself. Strictly speaking, I don't expect it to make a difference in the result of this test. 

It does, however, give a better visual indication of what's going on (where's the robot with respect to the couch table, etc.). Furthermore, it might be useful for the case where the robot computes the optimal pose for the base.